source/_posts/2017-11-27-Adding-Peek-and-Pop-to-your-View-Controllers.md

```diff
@@ -0,0 +1,145 @@
+---
+title:  Adding Peek and Pop to your View Controllers
+date: 2017-11-27 10:10:57
+tags: ios,peek,pop,3dtouch,forceTouchCapability
+authorIds:
+- dori
+categories:
+- iOS
+---
+
+With iOS 9, new iPhone models add a third dimension to the user interface.
+
+A user can now press your Home screen icon to immediately access functionality provided by your app.
+Within your app, a user can now press views to see previews of additional content and gain accelerated access to features.
```

Comment:
It might just be me, but ".. press views .." reads weird to me. Just wanted to make sure it isn't a typo?

Reply:
This is directly from apple